build.sbt

```diff
@@ -340,6 +340,9 @@ lazy val mtags3 = project
     unmanagedSourceDirectories.in(Compile) += baseDirectory
       .in(ThisBuild)
       .value / "mtags" / "src" / "main" / "scala",
+    managedSourceDirectories.in(Compile) += baseDirectory
+      .in(ThisBuild)
+      .value / "mtags" / "target" / ("scala-" + scalaVersion.value) / "src_managed" / "main",
```

Comment:
So I see this is added in order to access `BuildInfo` in `ScalaPresentationCompiler.scala`, but why isn't it detected by default here?

Reply:
It's the `mtags3` module, which I added so that we have the additional target. It uses `mtags3` directory by default, so every source dir from mtags needs to be added manually.